development-updates.md

```diff
@@ -112,7 +112,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
 | [Rodrigo Herrera](https://github.com/)                       | [Update 3](https://hackmd.io/@rodrigoh/update3)                 |                                                                 |                                                                 |                                                                                            |
 | [RoryArredondo](https://github.com/arredr2)                  | [Update 3](https://hackmd.io/@arredr2/Bkehft2IR)                | [Update 4](https://hackmd.io/@arredr2/BJ_GbXzDA)                |                                                                 |                                                                                            |
 | [Rose Jethani](https://github.com/rose2221)                  | [Update 3](https://hackmd.io/@0xrosetteeee/Week3)               | [Update 4](https://hackmd.io/@0xrosetteeee/Week4)               | [Update 5](https://hackmd.io/@0xrosetteeee/BJw067Eu0)           |                                                                                            |
-| [Rupam Dey](https://github.com/rupam-04)                     | [Update 3](https://hackmd.io/@rupam-04/Week3)                   | [Update 4](https://hackmd.io/@rupam-04/Week4)                   | [Update 5](https://hackmd.io/@rupam-04/Week5)                   |                                                                                            |
+| [Rupam Dey](https://github.com/rupam-04)                     | [Update 3](https://hackmd.io/@rupam-04/Week3)                   | [Update 4](https://hackmd.io/@rupam-04/Week4)                   | [Update 5](https://hackmd.io/@rupam-04/Week5)                   | [Light Client Support in Prysm](https://hackmd.io/@Bastin/BkdVcRQvA)                                                                                           |
```

Comment:
you should link the proposal within repo `projects/light-client-support-in-prysm.md`

Reply:
> you should link the proposal within repo `projects/light-client-support-in-prysm.md`

done!